### PR TITLE
fix timer, bug on exit ,add logging, add messagebox on exit

### PR DESCRIPTION
**Pull request by Ali2231**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -322,11 +326,14 @@ def quit(self):
...
-        self.ansTextBox.destroy()
-        self.congratz.destroy()
-        # input("Press any key to exit!")
 
+        self.quitGameMessage()
+        logging.debug('end of quit') 
+
+    def quitGameMessage(self):
+        showMessageBox("CorrectOrWrongMath","Thank you for playing! Good bye!")
+        g.destroy()
```
Comment (mnassar) [inline main.py:335]: is it better to use self.destroy() ? since g is defined outside the class?

File: main.py [hunk 2]
```diff
@@ -335,16 +342,25 @@ def play(self):
...
+    logging.getLogger().setLevel(logging.DEBUG)
 
 if __name__ == '__main__':
-
+    initLogging()
     # wave_obj = sa.WaveObject.from_wave_file(filename)
     # play_obj = wave_obj.play()
     g = game()
-
+    g.t_start = time.time()
```
Comment (mnassar) [inline main.py:360]: maybe not needed since we have self.t_start in run() ?

**Review by mnassar — CHANGES_REQUESTED** [review]
please check two comments

File: main.py (hunk 2 above)
Comment (Ali2231) [reply main.py:360]: for some reason the t_start is not initialized 
    minutes = (t_end - self.t_start) // 60
                       ^^^^^^^^^^^^
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.12_3.12.496.0_x64__qbz5n2kfra8p0\Lib\tkinter\__init__.py", line 2425, in __getattr__
    return getattr(self.tk, attr)
           ^^^^^^^^^^^^^^^^^^^^^^
AttributeError: '_tkinter.tkapp' object has no attribute 't_start'
adding this line was able to fix the bug.

**Pull request merged**